merlin/systems/dag/ensemble.py

```diff
@@ -21,9 +21,9 @@
 os.environ["PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION"] = "python"
 
 from google.protobuf import text_format  # noqa
+from merlin.dag import Graph  # noqa
```

Comment:
I think we should be adding merlin as a known first party package for isort (so it shouldn't be grouped with thirdparty things like google.protobuf).

something like https://github.com/NVIDIA-Merlin/models/pull/216

Reply:
(involves adding this line to the isort config https://github.com/NVIDIA-Merlin/models/blob/303a91eebb4859c7d4c5065f345b33e45962c69e/pyproject.toml#L21  )

Reply:
Added